src/mutable_data.rs

```diff
@@ -816,43 +869,46 @@ impl From<UnseqMutableData> for Data {
     }
 }
 
+/// Action for a Sequenced Entry.
 #[derive(Hash, Eq, PartialEq, PartialOrd, Ord, Clone, Serialize, Deserialize, Debug)]
 pub enum SeqEntryAction {
-    /// Inserts a new Sequenced entry
+    /// Inserts a new Sequenced entry.
     Ins(SeqValue),
-    /// Updates an entry with a new value and version
+    /// Updates an entry with a new value and version.
     Update(SeqValue),
-    /// Deletes an entry
+    /// Deletes an entry.
     Del(u64),
 }
 
+/// Action for an Unsequenced Entry.
 #[derive(Hash, Eq, PartialEq, PartialOrd, Ord, Clone, Serialize, Deserialize, Debug)]
 pub enum UnseqEntryAction {
-    /// Inserts a new Unsequenced entry
+    /// Inserts a new Unsequenced entry.
     Ins(Vec<u8>),
-    /// Updates an entry with a new value
+    /// Updates an entry with a new value.
     Update(Vec<u8>),
-    /// Deletes an entry
+    /// Deletes an entry.
     Del,
 }
 
-/// Helper struct to build entry actions on `MutableData`
+/// Sequenced Entry Actions for given entry keys.
 #[derive(Hash, Eq, PartialEq, PartialOrd, Ord, Clone, Serialize, Deserialize, Debug, Default)]
 pub struct SeqEntryActions {
     actions: BTreeMap<Vec<u8>, SeqEntryAction>,
 }
 
 impl SeqEntryActions {
-    /// Create a new Sequenced Entry Actions list
+    /// Creates a new Sequenced Entry Actions list.
     pub fn new() -> Self {
         Default::default()
     }
 
+    /// Gets the actions.
     pub fn actions(&self) -> &BTreeMap<Vec<u8>, SeqEntryAction> {
         &self.actions
     }
 
-    /// Insert a new key-value pair
+    /// Inserts a new key-value pair.
```

Comment:
It's also worth explaining what the `version` parameter means and what it should be equal to.